benchmarks/src/bench/find_many.rs

```diff
@@ -21,40 +19,43 @@ pub struct Options {
     pub uri: String,
 }
 
+#[async_trait::async_trait]
 impl Benchmark for FindManyBenchmark {
     type Options = Options;
 
-    fn setup(options: Self::Options) -> Result<Self> {
-        let client = Client::with_uri_str(&options.uri)?;
+    async fn setup(options: Self::Options) -> Result<Self> {
+        let client = Client::with_uri_str(&options.uri).await?;
         let db = client.database(&DATABASE_NAME);
-        db.drop()?;
+        db.drop(None).await?;
 
-        let mut file = File::open(options.path)?;
+        let num_iter = options.num_iter;
```

Comment:
Since the `spawn_blocking_and_await` macro I wrote implicitly wraps the contents in a `move` closure, we can't reuse `options` below. Since `num_iter` is just an int, it's simple to just copy beforehand.